cloudDetails/azure.json

```diff
@@ -109,35 +151,35 @@
       "Standard_E32as_v4": null,
       "Standard_E32as_v5": {
         "roachprodArgs": {
+          "azure-volume-size": "1000",
```

Comment:
We should keep 2.5tb for this one, since it's vcpu32 type.

Reply:
Thank you for pointing it out! Deleted this line.